x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/ml/dataframe/analyses/Classification.java

```diff
@@ -218,7 +218,7 @@ public Classification(StreamInput in) throws IOException {
         } else {
             featureProcessors = Collections.emptyList();
         }
-        if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
+        if (in.getVersion().onOrAfter(Version.V_7_12_0)) {
```

Comment:
You can remove this check entirely as `master` may only talk to 7.latest.

Reply:
Fixed in 68edfa91a00